metadata/it.json

```diff
@@ -12,5 +13,11 @@
 		"nameserver.cnr.it",
 		"r.dns.it",
 		"s.dns.it"
+	],
+	"policies": [
+		{
+			"type": "idn-table",
+			"comment": "http://www.crdd.it/norme/GuidelineTecnicheSincrono2.1-EN.pdf"
```

Comment:
What’s up with this?

Reply:
This is the best I could find, section 2.2 in this PDF file. The info isn't on their website.